setup.py

```diff
@@ -375,8 +375,8 @@ def parse_setuppy_commands():
     # useful messages to the user
     if 'install' in args:
         print(textwrap.dedent("""
-            Note: if you need reliable uninstall behavior, then install
-            with pip instead of using `setup.py install`:
+            Note: for reliable dependency installation and uninstallation
```

Comment:
Can you keep "uninstall" first? I think that's the more important case, and it helps clarify it's about uninstalling NumPy, not uninstalling dependencies.